app/assets/stylesheets/layouts/homepage/l-featured-insights.scss

```diff
@@ -1,7 +1,7 @@
 .l-featured-insights {
   margin: 0 0 40px;
 
-  .title {
```

Comment:
All `.title` in the homepage has the same `margin-bottom` (40px), what's the point of renaming the class?

Reply:
By scoping all `.title`s it has the unintended consequence of changing the title components in the cards, which use the same `.title` class. I introduced a new component name.